packages/design/.storybook/main.ts

```diff
@@ -18,6 +18,7 @@ const config: StorybookConfig = {
     getAbsolutePath('@storybook/addon-interactions'),
     getAbsolutePath('@storybook/addon-a11y'),
     getAbsolutePath('@storybook/addon-coverage'),
+    getAbsolutePath('@storybook/experimental-addon-test'),
```

Comment:
This will hopefully come out of "experimental" status soon, but seems to be running pretty smoothly now.